### PR TITLE
Feature/#619 pin code system tab is white on i os

### PR DESCRIPTION
**Pull request by YaroTs**

### CODE REVIEW

File: mobile/src/splash-screen/splash-screen.tsx [hunk 1]
```diff
@@ -91,6 +91,17 @@ class SplashScreenImpl extends React.Component<SplashScreenStateProps & SplashSc
 
     //----------------------------------------------------------------------------
     public render() {
+        const splashContent = this.renderContent();
```
Comment (antonm76) [inline mobile/src/splash-screen/splash-screen.tsx]: Why do you need `splashContent` variable here? Isn't it possible to call `this.renderContent()` inline?

**Comment by DesmanLead** [comment]
Now there is a blue stripe at the top, right?
Did you consider setting the status bar text color to black?

**Comment by YaroTs** [comment]
Yes, it is. Why the text color should be black?

**Comment by antonm76** [comment]
Just make two screenshots and ask @KuchaLeni what is better.

**Comment by DesmanLead** [comment]
I mean, don't add a blue stripe. Just black text on the white background.

File: mobile/src/splash-screen/splash-screen.tsx (hunk 1 above)
Comment (YaroTs) [reply mobile/src/splash-screen/splash-screen.tsx]: Not sure how to do it correctly. Such code
```
<SafeAreaView style={splashScreenStyles.container}>
                {this.renderContent()}
            </SafeAreaView>
```
produce Top-level 'this' expression warning

**Pull request merged**